test/letters.js

```diff
@@ -0,0 +1,240 @@
+var fs      = require('fs');
+var chai    = require('chai');
+var expect  = chai.expect;
+
+var API_KEY = 'test_0dc8d51e0acffcb1880e0f19c79b2f5b0cc';
+var Lob     = require('../lib/index.js')(API_KEY);
+
+describe('Letters', function () {
+
+  describe('list', function () {
+    it('should error with an invalid count or offset', function (done) {
+      Lob.letters.list({ offset: 0,count: 1000 }, function (err) {
+        expect(err).to.exist;
+        done();
+      });
+    });
+
+    it('should have the correct defaults', function (done) {
+      Lob.letters.list(function (err, res) {
+        expect(res).to.have.property('object');
+        expect(res).to.have.property('data');
+        expect(res.data).to.be.instanceof(Array);
+        expect(res.data.length).to.eql(10);
+        expect(res).to.have.property('count');
+        expect(res).to.have.property('next_url');
+        expect(res.next_url).to.eql('https://api.lob.com/' +
+          'v1/letters?count=10&offset=10');
+        expect(res).to.have.property('previous_url');
+        expect(res.object).to.eql('list');
+        expect(res.count).to.eql(10);
```

Comment:
I was mostly just following the tests for postcards and this is how it was done. `expect` doesn't take a long to to execute, right? so i don't see the harm in making sure all of it is there